rootfs/deploy.py

```diff
@@ -0,0 +1,49 @@
+import docker
+import os
+import tarfile
+import os.path
+import boto3
+import requests
+
+from botocore.client import Config
+from urllib.parse import urlparse
+
+
+def log_output(stream):
+    for chunk in stream:
+        print(chunk)
+
+
+if os.getenv('TAR_URL'):
```

Comment:
Nitpick: This sections calls `os.getenv('TAR_URL')` three times. This would be cleaner:
```python
tar_url = os.getenv('TAR_URL')
if tar_url:
    if os.path.exists("/var/run/secrets/object/store/access-key-id"):
        ....
        url = urlparse(tar_url)
        ...
    else:
        r = requests.get(tar_url)
        ....
```